google/cloud/BUILD

```diff
@@ -55,8 +55,11 @@ load(
 [cc_test(
     name = "google_cloud_cpp_common_" + test.replace("/", "_").replace(".cc", ""),
     srcs = [test],
-    # TODO(#664 / #666) - use the right condition when porting Bazel builds
-    linkopts = ["-lpthread"],
+    # TODO(#664) - use the right condition when porting Bazel builds
+    linkopts = select({
+        "@bazel_tools//src/conditions:linux_x86_64": ["-lpthread"],
```

Comment:
What do you think about inverting the conditions, since `pthread` is used on more than just Linux? Using platform (untested), something like:

```
":windows": [""],
"//conditions:default": ["-lpthread"],
```

Note I suggest using the platform constraint here since it is more generic.

Reply:
I inverted the condition.

"@bazel_tools//src/conditions:windows" is defined at:
https://github.com/bazelbuild/bazel/blob/ec7084d91ae2ac20102729e13d6f1e2ad442f81d/src/conditions/BUILD.tools#L31

If I use `":windows":` in select, I needed to have the definition of it in the same file. I think we can just use "@bazel_tools//src/conditions:". WDYT?

Reply:
Sorry, I didn't mean `:windows` directly. I meant something like `@platforms//os:windows` per the [docs](https://docs.bazel.build/versions/master/platforms.html). I am not familiar enough with Bazel to know if this is more generic than `@bazel_tools//src/conditions:windows`.

Reply:
Probably we can just use `@bazel_tools//src/conditions`. Anyways thanks for the suggestion. With the inverted condition, it's slightly better now.